ctapipe_io_lst/pointing.py

```diff
@@ -55,10 +62,57 @@ def _read_drive_report(path):
                 'Az', 'azimuth_avg', 'azimuth_min', 'azimuth_max', 'azimuth_std',
                 'El', 'zenith_avg', 'zenith_min', 'zenith_max', 'zenith_std',
```

Comment:
If I get it right, this will work aso when the 'El' tag in the file is changed to the correct 'Zd'.  If so, we can just put here the new label, right? (to be confirmed with the Drive team)

Reply:
The name here does not play any role, I could also have added "_" or "foo" or someting else. It is ignored and not included in the output table (see `excluded_names`)

Reply:
Ok, so just give it the new name, or a dummy, to remove this wrong "elevation" reference which will soon disappear from the files as well.